packages/frameworks-express/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@auth/express",
+  "description": "Authentication for Express.",
+  "version": "0.0.1",
+  "type": "module",
+  "files": [
+    "index.*",
+    "http-api-adapters.*",
+    "src"
+  ],
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    },
+    "./http-api-adapters": {
+      "types": "./http-api-adapters.d.ts",
+      "import": "./http-api-adapters.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "tsup --config ./tsup.config.js && node scripts/postbuild",
+    "patch": "npm version patch --no-git-tag-version",
+    "clean": "rm -rf index.*",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --config ./jest.config.cjs"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@auth/core": "workspace:*",
+    "@jest/globals": "^29.6.3",
+    "@types/express": "^4.17.17",
+    "@types/node": "^18.7.14",
+    "@types/node-fetch": "^2.6.4",
+    "@types/supertest": "^2.0.12",
+    "express": "^4.18.2",
+    "jest": "^29.6.2",
+    "node-fetch": "^2.6.12",
+    "supertest": "^6.3.3",
+    "ts-jest": "29.0.0-next.1",
+    "tsup": "^6.5.0",
```

Comment:
Although tempting, let's only use `tsc` to not introduce unnecessary tools to the monorepo without doing it holistically